projects/knora-ui/ng-package.json

```diff
@@ -2,6 +2,13 @@
   "$schema": "../../node_modules/ng-packagr/ng-package.schema.json",
   "dest": "../../dist/knora-ui",
   "lib": {
-    "entryFile": "src/public-api.ts"
+    "entryFile": "src/public-api.ts",
+    "umdModuleIds": {
+      "@knora/api": "@knora/api",
+      "ngx-color-picker": "ngxColorPicker",
+      "jdnconvertiblecalendar": "jdnconvertiblecalendar",
+      "jdnconvertiblecalendardateadapter": "jdnconvertiblecalendardateadapter",
+      "jdnconvertiblecalendar/dist/src/JDNConvertibleCalendar": "JDNConvertibleCalendar"
```

Comment:
I wonder why there is `jdnconvertiblecalendar` and `JDNConvertibleCalendar`.  `jdnconvertiblecalendar/dist/src/JDNConvertibleCalendar` seems odd. Maybe I did something wrong in https://github.com/dhlab-basel/JDNConvertibleCalendar, I will check this.